app/models/requirement.rb

```diff
@@ -1,10 +1,27 @@
 class Requirement < ActiveRecord::Base
+  include AASM
   belongs_to :deliverable
   validates :name, { presence: true }
   validates :deliverable, { presence: true }
 
   before_validation :set_defaults
 
+  enum status: [:created, :started, :completed]
+
+  aasm column: :status do
```

Comment:
Do we need a several states for this -- what about inferring 'started' based on whether someone has picked it up and just having a 'completed' boolean?